app/controllers/application_controller/performance.rb

```diff
@@ -206,12 +206,12 @@ def perf_set_or_fix_dates(options, allow_interval_override = true)
       options[:typ] = "Hourly" if options[:typ] == "Daily" && edate_daily < sdate_daily
     end
 
-    if options[:hourly_date] && # Need to clear hourly date if not nil so it will be reset below if
+    if options[:hourly_date].present? && # Need to clear hourly date if not nil so it will be reset below if
```

Comment:
@simon3z @zeari a better test will be `options[:hourly_date].to_date.present?` ?
because this test is guarding the line: `options[:hourly_date].to_date < sdate.to_date`